mlflow/experiments.py

```diff
@@ -4,10 +4,11 @@
 
 import click
 from tabulate import tabulate
+import pandas as pd
```

Comment:
This is redundant and fails the linting test. you can just remove it.

Reply:
Done